README.md

```diff
@@ -154,8 +154,17 @@ Example Playbook
                   format: 'qcow2'
                   capacity: '200GB'
                   pool: 'my-pool'
+                - name: 'filestore'
+                  type: 'file'
+                  capacity: '900GB'
               interfaces:
-                - network: 'br-datacentre'
```

Comment:
We've lost this example of type=network which should still work.

Reply:
it should still appear in the 'vm1' example further up (~ line 145)